module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md

```diff
@@ -48,7 +48,7 @@ Other services can use the information stored in the ServiceEndpoint entity to f
 ```powershell
 Connect-Entra -Scopes 'Application.Read.All'
 $application = Get-EntraApplication -Filter "DisplayName eq 'Contoso Helpdesk Application'"
-Get-EntraApplicationServiceEndpoint -ApplicationId $application.ObjectId
+Get-EntraApplicationServiceEndpoint -ApplicationId $application.Id
```

Comment:
Fails with an error: `A parameter cannot be found that matches parameter name 'ApplicationId'.`
```suggestion
Get-EntraApplicationServiceEndpoint -ObjectId $application.Id
```